src/networkremote/networkremote.cpp

```diff
@@ -213,6 +219,11 @@ void NetworkRemote::CreateRemoteClient(QTcpSocket* client_socket) {
     RemoteClient* client = new RemoteClient(app_, client_socket);
     clients_.push_back(client);
 
+    // Update the Remote Root Files for the latest Client
+    outgoing_data_creator_->SetMusicEextensions(client->music_extensions());
```

Comment:
`SetMusicExtensions`

Reply:
arf... it was late when I finished this part. sorry...